models/br_bd_diretorios_brasil/code/cid_10.py

```
@@ -0,0 +1,376 @@
+        "causa_violencia",
+        "causa_overdose",
+        "indicador_registro_ativo",
+    ]
+
+
+# DATASUS
+def download_datasus(input_dir: Path) -> None:
+    """Download and unzip the DATASUS CID-10 CSV package into `input_dir`."""
+    input_dir.mkdir(parents=True, exist_ok=True)
+
+    response = requests.get(
```

Ruff (0.16.1)

[error] Probable use of `requests` call without timeout

(S113)